tests/integration.rs

```diff
@@ -87,22 +87,23 @@ async fn main() {
     );
 
     // Construct a settings configuration interface.
-    let mut handlers = [None; 10];
-    let mut interface: minireq::Minireq<bool, _, _, minimq::broker::IpBroker> =
-        minireq::Minireq::new("minireq/integration/device", mqtt, &mut handlers[..]).unwrap();
-
-    interface
-        .register("test", |exit, _req, _data, _out_buffer| {
-            *exit = true;
-            Ok(0)
-        })
-        .unwrap();
+    let mut interface: minireq::Minireq<_, _, minimq::broker::IpBroker> =
+        minireq::Minireq::new("minireq/integration/device", mqtt).unwrap();
+
+    interface.register("test").unwrap();
```

Comment:
Maybe call this `subscribe()` now.

Reply:
Fixed in 524a9c12b2e00312ef16212b0610fd8fac261314